lib/Models/CatalogItem.js

```diff
@@ -318,6 +327,20 @@ CatalogItem.defaultUpdaters.rectangle = function(catalogItem, json, propertyName
     }
 };
 
+CatalogItem.defaultUpdaters.textAttribution = function(catalogItem, json, prototypeName) {
+    if(defined(json.textAttribution)){
+        if((typeof json.textAttribution === 'object' && json.textAttribution[0].text) && json.textAttribution[0].link ){
```

Comment:
Why the array inside `textAttribution`?  Shouldn't it just be `textAttribution.text` (and then update the json file accordingly)?